admin/product.php

```diff
@@ -97,13 +97,29 @@
     <!-- body_text //-->
     <?php
     if ($action == 'new_product_meta_tags') {
-      require(DIR_WS_MODULES . $zc_products->get_handler($product_type) . '/collect_info_metatags.php');
+      if (file_exists(DIR_WS_MODULES . $zc_products->get_handler($product_type) . '/collect_info_metatags.php')) {
```

Comment:
it seems all of this code repetition is ripe for a refactor.  could we not create a function like:

```
function zenRequireFile($fileName)
    {
        global $zc_products, $product_type;
        if (file_exists(DIR_WS_MODULES . $zc_products->get_handler($product_type) . '/' . $fileName)) {
            require(DIR_WS_MODULES . $zc_products->get_handler($product_type) . '/' . $fileName);
        } else {
            require(DIR_WS_MODULES . $fileName);
        }
    }
```
and then call that function?  ie:

```
zenRequireFile('update_product_meta_tags.php');
```
i'm seeing 7 times the exact same code is being called within this script.  unless i am missing something...

Reply:
I was okay with that initially since it was contained to just the one file. 
But then I found it also in one other file. Granted, in only 1 other file.
But yes, function added.